---
title: "Growth yields and light-capture in PhycoCyanin and PhycoErythrin-rich picocyanobacteria, across photic regimes and growth phases"

author:
- Sylwia Śliwińska-Wilczewska:
    institute:
    - MTA
    - UG
    email: ssliwinskawilczews@mta.ca
    ORCID: ORCID 0000-0002-3147-6605
- Marta Konik:
    institute:
    - VU
    - IOPAN
    email: mk@iopan.gda.pl
    ORCID: 0000-0003-1145-9127
- Mireille Savoie:
    institute: MTA
    email: msavoie@mta.ca
    ORCID: 0009-0009-9499-6657
- Anabella Aguilera:
    institute: 
    - SW
    - SW2
    email: anabella.aguilera@lnu.se
    ORCID: 0000-0001-6743-3001
- Naaman M. Omar:
    institute: MTA
    email: nomar@mta.ca
    ORCID: 0000-0001-9583-2886
- Douglas A. Campbell:
    institute: MTA
    email: dubhglascambeuil@gmail.com
    ORCID: 0000-0001-8996-5463
    correspondence: true
institute:
- MTA: Department of Biology, Mount Allison University, 53 York St., Sackville, NB, E4L 1C9, Canada
- UG: "Institute of Oceanography, University of Gdansk, 46 Pilsudskiego St, P81-378,
    Gdynia, Poland"
- VU: Department of Geography, University of Victoria, Victoria, BC, V8P 5C2, Canada
- IOPAN: "Institute of Oceanology, Polish Academy of Sciences, 81-712 Sopot, Poland"
- SW: Department of Biology and Environmental Science, Centre for Ecology and Evolution in Microbial Model Systems (EEMiS), Linnaeus University, Kalmar, Sweden
- SW2: "Department of Aquatic Sciences and Assessment, Swedish University of Agricultural Sciences, Uppsala, Sweden (present address)"
output:
  bookdown::word_document2:
    reference_docx: Template.docx
    code-folding: show
    keep_md: yes
    fig.caption: yes
    toc: FALSE
    pandoc_args:
    - "--lua-filter=scholarly-metadata.lua"
    - "--lua-filter=author-info-blocks.lua"
  html_document:
    df_print: paged
  word_document:
    reference_docx: Template.docx
    code-folding: show
    keep_md: yes
    fig.caption: yes
    toc: FALSE
    pandoc_args:
    - "--lua-filter=scholarly-metadata.lua"
    - "--lua-filter=author-info-blocks.lua"
    - "--lua-filter=custom_filter.lua"
bibliography:
- BalticPhotoperiod.bib
- packages.bib
- faultycitations.bib
csl: "limnology-and-oceanography.csl"
---

```{r, import libraries, include = FALSE}
library(tidyverse)
library(glue)
library(googlesheets4)
googlesheets4:: gs4_deauth()
#library(kableExtra)
library(flextable)
library(officer)
library(knitr)
library(readxl)
library(stringr)
library(rmarkdown)
library(gdtools)
#library(trackdown)
options(knitr.table.format = "word_document2", knitr.kable.NA = '') 
```

```{r, save package citations, include = FALSE}
knitr:: write_bib(c("tidyverse", "glue", "googlesheets4", "kableExtra"), "packages.bib")
```

```{r defining filepaths, include = FALSE}
FiguresFP <- file.path("..", "Output", "Figures")
```

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

<br>

# Supporting Information {.unnumbered}

```{r locally stored metadata from rds, echo = FALSE}
# https://davidgohel.github.io/flextable/reference/index.html

# BalticPhotoperiod_Tab_PigmentsCorrelation.Rds
# BalticPhotoperiod_Tab_GrowthCorrelation.Rds

# BalticPhotoperiod_Anova_GrowthRate.Rds
# BalticPhotoperiod_Anova_GrowthRate_Model_Light.Rds
# BalticPhotoperiod_Anova_GrowthRate_Model_Photoperiod.Rds

# BalticPhotoperiod_Anova_GS.Rds
# BalticPhotoperiod_Anova_GS_Model.Rds

# BalticPhotoperiod_Anova_PhycoChlaRatio.Rds
# BalticPhotoperiod_Anova_PhycoChlaRatio_Model.Rds

# BalticPhotoperiod_Anova_PURPARRatio.Rds
# BalticPhotoperiod_Anova_PURPARRatio_Model.Rds

# BalticPhotoperiod_Anova_Sig445.Rds
# BalticPhotoperiod_Anova_Sig445_Model.Rds

# BalticPhotoperiod_Anova_Sig590.Rds
# BalticPhotoperiod_Anova_Sig590_Model.Rds

# BalticPhotoperiod_Anova_SigPig445.Rds
# BalticPhotoperiod_Anova_SigPig445_Model.Rds

# BalticPhotoperiod_Anova_SigPig590.Rds
# BalticPhotoperiod_Anova_SigPig590_Model.Rds

```

<br>
<br>
<br>

```{r echo = FALSE}
PigmentsCorrelation <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Tab_PigmentsCorrelation.Rds"))
PigmentsCorrelation<-PigmentsCorrelation %>%  
  relocate(Pigment, .before = Linear_regression) %>%
  relocate(Abs, .before = Linear_regression)

myft <- flextable(PigmentsCorrelation)
myft <- set_caption(myft, caption = "Linear regression, coefficient of determination (R square), Pearson correlation coefficients (R), and p-value used to calculate the pigment content (µg mL-1) of two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) (Culture Collection of Baltic Algae) of Synechococcus sp. based on absorbance (A) measurements.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(PigmentsCorrelation)
```

<br>

```{r echo = FALSE}
Anova_GrowthRate <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_GrowthRate.Rds"))
Anova_GrowthRate<-Anova_GrowthRate
df <- data.frame(Source_of_variation = c("Par_ue", "Photoperiod", "Strain", "Par_ue:Photoperiod", "Par_ue:Strain", "Photoperiod:Strain", "Par_ue:Photoperiod:Strain", "Residuals"))
Anova_GrowthRate<-cbind(df, Anova_GrowthRate) 
Anova_GrowthRate<-Anova_GrowthRate %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  mutate(Df=as.character(Df))


myft <- flextable(Anova_GrowthRate) 
myft <- set_caption(myft, caption = "Three-way factorial ANOVA testing whether peak PAR, photoperiod, strain, and their interactions (Source_of_variation), significantly influence the chlorophyll specific exponential growth rate (µ; d−1), estimated from logistic fits of chlorophyll proxy OD680 – OD720 vs. cumulative diel PUR, for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) (Culture Collection of Baltic Algae) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Df – degrees of freedom; Sum Sq – sum of squares; Mean Sq – mean sum of squares; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_GrowthRate, df)
```

<br>

```{r echo = FALSE}
GrowthRate_Model_Photoperiod <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_GrowthRate_Model_Photoperiod_PUR.Rds"))
GrowthRate_Model_Photoperiod<-GrowthRate_Model_Photoperiod %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  relocate(Strain, .before = Fit_model) %>% 
  mutate(Res.Df=as.character(Res.Df))
  
myft <- flextable(GrowthRate_Model_Photoperiod)
myft <- set_caption(myft, caption = "One-way ANOVA of a three parameter model (Harrison and Platt 1986) from pooled data (All) and data fit across different photoperiods (8, 12, 16, or 24) from chlorophyll specific exponential growth rate vs. cumulative diel PUR (Fit_model), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(GrowthRate_Model_Photoperiod)
```

<br>

```{r echo = FALSE}
Anova_GrowthRate_Model_Light <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_GrowthRate_Model_Light_PUR.Rds"))
Anova_GrowthRate_Model_Light<-Anova_GrowthRate_Model_Light %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  relocate(Strain, .before = Fit_model) %>% 
  mutate(Res.Df=as.character(Res.Df))


myft <- flextable(Anova_GrowthRate_Model_Light)
myft <- set_caption(myft, caption = "One-way ANOVA of a three parameter model (Harrison and Platt 1986) from pooled data (All) and data fit across different peak PAR (30, 90, 180, 300, 600 together with 900) from chlorophyll specific exponential growth rate vs. cumulative diel PUR (Fit_model), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_GrowthRate_Model_Light)
```

<br>

```{r echo = FALSE}
GrowthRate_Model_PhotoperiodPAR <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_GrowthRate_Model_Photoperiod.Rds"))
GrowthRate_Model_PhotoperiodPAR<-GrowthRate_Model_PhotoperiodPAR %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  relocate(Strain, .before = Fit_model) %>% 
  mutate(Res.Df=as.character(Res.Df))
  
myft <- flextable(GrowthRate_Model_PhotoperiodPAR)
myft <- set_caption(myft, caption = "One-way ANOVA of a three parameter model (Harrison and Platt 1986) from pooled data (All) and data fit across different photoperiods (8, 12, 16, or 24) from chlorophyll specific exponential growth rate vs. cumulative diel PAR (Fit_model), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(GrowthRate_Model_PhotoperiodPAR)
```

<br>

```{r echo = FALSE}
Anova_GrowthRate_Model_LightPAR <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_GrowthRate_Model_Light.Rds"))
Anova_GrowthRate_Model_LightPAR<-Anova_GrowthRate_Model_LightPAR %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  relocate(Strain, .before = Fit_model) %>% 
  mutate(Res.Df=as.character(Res.Df))


myft <- flextable(Anova_GrowthRate_Model_LightPAR)
myft <- set_caption(myft, caption = "One-way ANOVA of a three parameter model (Harrison and Platt 1986) from pooled data (All) and data fit across different peak PAR (30, 90, 180, 300, 600 together with 900) from chlorophyll specific exponential growth rate vs. cumulative diel PAR (Fit_model), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_GrowthRate_Model_LightPAR)
```

<br>

```{r echo = FALSE}
GrowthRate_Model_PhotoperiodJVPSII <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_GrowthRateJVPSII_Model_Photoperiod_Chla.Rds"))
GrowthRate_Model_PhotoperiodJVPSII<-GrowthRate_Model_PhotoperiodJVPSII %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  relocate(Strain, .before = Fit_model) %>% 
  mutate(Res.Df=as.character(Res.Df))
  
myft <- flextable(GrowthRate_Model_PhotoperiodJVPSII)
myft <- set_caption(myft, caption = "One-way ANOVA of a three parameter model (Harrison and Platt 1986) from pooled data (All) and data fit across different photoperiods (8, 12, 16, or 24) from chlorophyll specific exponential growth rate vs. PSII electron flux (JVPSII; µmol e− µmol Chl a−1 d−1) (Fit_model), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(GrowthRate_Model_PhotoperiodJVPSII)
```

<br>

```{r echo = FALSE}
Anova_GrowthRate_Model_LightJVPSII <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_GrowthRateJVPSII_Model_Light_Chla.Rds"))
Anova_GrowthRate_Model_LightJVPSII<-Anova_GrowthRate_Model_LightJVPSII %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  relocate(Strain, .before = Fit_model) %>% 
  mutate(Res.Df=as.character(Res.Df))


myft <- flextable(Anova_GrowthRate_Model_LightJVPSII)
myft <- set_caption(myft, caption = "One-way ANOVA of a three parameter model (Harrison and Platt 1986) from pooled data (All) and data fit across different peak PAR (30, 90, 180, 300, 600 together with 900) from chlorophyll specific exponential growth rate vs. PSII electron flux (JVPSII; µmol e− µmol Chl a−1 d−1) (Fit_model), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_GrowthRate_Model_LightJVPSII)
```

<br>

```{r echo = FALSE}
Anova_PURPARRatio_Model <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_PURPARRatio_Model.Rds"))
Anova_PURPARRatio_Model<-Anova_PURPARRatio_Model %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  mutate(Res.Df=as.character(Res.Df))
  

myft <- flextable(Anova_PURPARRatio_Model)
myft <- set_caption(myft, caption = "One-way ANOVA of single phase exponential decay fit model (Fit_model) of pooled data across different strains for a given phase of growth (exponential; _Exp, pre-stationary; _St) and across different phase of growth for a given strain (_Exp_St) from PUR/PAR ratio in relation to the cumulative diel PAR (µmol photons m−2d−1), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_PURPARRatio_Model)
```

<br>

```{r echo = FALSE}
Anova_PhycoChlaRatio_Model <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_PhycoChlaRatio_Model.Rds"))
Anova_PhycoChlaRatio_Model<-Anova_PhycoChlaRatio_Model %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  mutate(Res.Df=as.character(Res.Df))
  

myft <- flextable(Anova_PhycoChlaRatio_Model)
myft <- set_caption(myft, caption = "One-way ANOVA of single phase exponential decay fit model (Fit_model) of pooled data across different strains for a given phase of growth (exponential; _Exp, pre-stationary; _St) and across different phase of growth for a given strain (_Exp_St) from Phycobiliprotein to Chl a ratio in relation to the cumulative diel PAR (µmol photons m−2d−1), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_PhycoChlaRatio_Model)
```

<br>

```{r echo = FALSE}
Anova_Sig590_Model <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_Sig590_Model.Rds"))
Anova_Sig590_Model<-Anova_Sig590_Model %>% 
  select(-c("Df", "Sum Sq")) %>% 
  rename(p_value = "Pr(>F)") %>% 
  rename(F_value = "F value") %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Res.Df) %>% 
  mutate(Res.Df=as.character(Res.Df))
  

myft <- flextable(Anova_Sig590_Model)
myft <- set_caption(myft, caption = "One-way ANOVA of single phase exponential decay fit model (Fit_model) of pooled data across different strains for a given phase of growth (exponential; _Exp, pre-stationary; _St) and across different phase of growth for a given strain (_Exp_St) from effective absorption cross section of PSII (σPSII′; nm2 quanta−1) measured under diel peak PAR growth light under Ex590nm (orange) excitation in relation to the cumulative diel PAR (µmol photons m−2d−1), for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Res.Df - residual degrees of freedom for each model; Res.Sum Sq - residual sum of squares for each model; F_value – Fisher's F-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_Sig590_Model)
```

<br>

```{r echo = FALSE}
Anova_Sig445_Model <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_Sig445_Model.Rds"))
Anova_Sig445_Model<-Anova_Sig445_Model %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Estimate) 


myft <- flextable(Anova_Sig445_Model)
myft <- set_caption(myft, caption = "T-test of linear fit model (Fit_model) of pooled data across different strains for a given phase of growth (exponential; _Exp, pre-stationary; _St) and across different phase of growth for a given strain (_Exp_St) from effective absorption cross section of PSII (σPSIIʹ; nm2 quanta−1) measured under diel peak PAR growth light under Ex445nm (blue) excitation in relation to the cumulative diel PAR (µmol photons m−2d−1, for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Estimate - estimation statistics; Std.Error - standard error of the estimate; t_value – t-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- set_formatter(myft, Estimate = function(x) {formatC(x, format = "e", digits = 3)})
myft <- set_formatter(myft, Std.Error = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_Sig445_Model)
```

<br>

```{r echo = FALSE}
Anova_SigPig445_Model <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_SigPig445_Model.Rds"))
Anova_SigPig445_Model<-Anova_SigPig445_Model %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Estimate) 
  

myft <- flextable(Anova_SigPig445_Model)
myft <- set_caption(myft, caption = "T-test of linear fit model (Fit_model) of pooled data across different strains for a given phase of growth (exponential; _Exp, pre-stationary; _St) and across different phase of growth for a given strain (_Exp_St) from effective absorption cross section of PSII (σPSIIʹ; nm2 quanta−1) measured under diel peak PAR growth light under Ex445nm (blue) excitation in relation to Phycobiliprotein to Chl a ratio, for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Estimate - estimation statistics; Std.Error - standard error of the estimate; t_value – t-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_SigPig445_Model)
```

<br>

```{r echo = FALSE}
Anova_SigPig590_Model <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_SigPig590_Model.Rds"))
Anova_SigPig590_Model<-Anova_SigPig590_Model %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Estimate) 
  

myft <- flextable(Anova_SigPig590_Model)
myft <- set_caption(myft, caption = "T-test of linear fit model (Fit_model) of pooled data across different strains for a given phase of growth (exponential; _Exp, pre-stationary; _St) and across different phase of growth for a given strain (_Exp_St) from effective absorption cross section of PSII (σPSII′; nm2 quanta−1) measured under Ex590nm (orange) excitation in relation to the Phycobiliprotein to Chl a ratio, for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Estimate - estimation statistics; Std.Error - standard error of the estimate; t_value – t-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_SigPig590_Model)
```

<br>

```{r echo = FALSE}
Anova_SigPig590_Modeldark <- readRDS(file = file.path("..", "Output", "TablesRds", "BalticPhotoperiod_Anova_SigPig590dark_Model.Rds"))
Anova_SigPig590_Modeldark<-Anova_SigPig590_Modeldark %>% 
  rename(Fit_model = "FitComparison") %>% 
  relocate(Fit_model, .before = Estimate) 
  

myft <- flextable(Anova_SigPig590_Modeldark)
myft <- set_caption(myft, caption = "T-test of linear fit model (Fit_model) of pooled data across different strains for a given phase of growth (exponential; _Exp, pre-stationary; _St) and across different phase of growth for a given strain (_Exp_St) from effective absorption cross section of PSII (σPSII; nm2 quanta−1) measured under Ex590nm (orange) excitation in relation to the Phycobiliprotein to Chl a ratio, for two PhycoCyanin(PC)-rich cultures (056, 077) and two PhycoErythrin(PE)-rich cultures (048, 127) of Synechococcus sp., grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m−2s−1; and photoperiods of 8, 12, 16, or 24 h. Estimate - estimation statistics; Std.Error - standard error of the estimate; t_value – t-test statistic; p_value - level of significance.")
myft <- colformat_double(x = myft, big.mark = ",", digits = 3, na_str = "N/A")
myft <- set_formatter(myft, F_value = function(x) {formatC(x, format = "e", digits = 3)})
myft <- fontsize(myft, size = 12, part = "header")
myft <- fontsize(myft, size = 10, part = "body")
myft <- align(myft, align = "left", part = "all")
myft <- autofit(myft)
myft

rm(Anova_SigPig590_Modeldark)
```

<br>

```{r Genetics, fig.cap = Genetics_cap, fig.height = 8, fig.width = 6, echo = FALSE}
Genetics_cap<- glue("Phylogenetic tree derived from partial 16S rRNA gene sequences using topology given by Maximum Likelihood (1000 bootstraps). Support values are indicated by the size of internal nodes. Strains used in this study are shown in bold.", .sep = " ")
knitr::include_graphics(file.path(FiguresFP, "SFig.Tree.png"))
```

<br>

```{r GrowthCurveOD680, fig.cap = GrowthCurveOD680_cap, fig.height = 8, fig.width = 6, echo = FALSE}
GrowthCurveOD680_cap<- glue("Growth curves, tracked as OD~680~ (**A**) and OD~720~ (**B**) vs. elapsed time (d). Growth curves were estimated over 5-min intervals for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m^−2^s^−1^; and photoperiods of 8, 12, 16, or 24 h. The vertical lines represent the time when the cultures reached the maximum of the 1^st^ derivative of OD~680~, or maximum absolute hourly growth (tMaxAHG), taken as an index of transition from exponential to pre-stationary growth phases. The orange area represents the photoperiods, with peak PAR x 1/1000 to scale to the Y axis.", .sep = " ")
knitr::include_graphics(file.path(FiguresFP, "SFig_GrowthCurve_OD680720.png"))
```

<br>

```{r GrowthCurveDeltaOD, fig.cap = GrowthCurveDeltaOD_cap, fig.height = 8, fig.width = 6, echo = FALSE}
GrowthCurveDeltaOD_cap<- glue("(**A**) Growth curves (tracked as chlorophyll proxy OD~680~-OD~720~; \u0394 OD) vs. elapsed time (d). Growth curves were estimated over 5-min intervals for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30, 90, 180, 300, 600, or 900 peak PAR µmol photons m^−2^s^−1^; and photoperiods of 8, 12, 16, or 24 h. The vertical lines represent the time when the cultures reached the maximum of the 1^st^ derivative of OD~680~, or maximum absolute hourly growth (tMaxAHG), taken as an index of transition from exponential to pre-stationary growth phases. The orange area represents the photoperiods, with peak PAR x 1/2000 to scale to the Y axis. (**B**) Logistic fits (thick lines) of chlorophyll proxy OD~680~-OD~720~ (\u0394 OD) vs. elapsed time (d). Growth curves (thin line) measured over 5-min intervals for each strain were also presented.", .sep = " ")
knitr::include_graphics(file.path(FiguresFP, "SFig_GrowthCurve_ODDeltaFit.png"))
```

<br>

```{r GrowthRateLightPUR, fig.cap = GrowthRateLightPUR_cap, fig.height = 8, fig.width = 6, echo = FALSE}
GrowthRateLightPUR_cap<- glue("(**A**) Chlorophyll-specific exponential growth rates (d^−1^) vs. cumulative diel Photosynthetically Usable Radiation (PUR, µmol photons m^−2^d^−1^). Growth rates (± SE falling within symbols) were estimated from logistic fits of chlorophyll proxy OD~680~ – OD~720~ (ΔOD) vs. elapsed time (Fig. 1, Fig. S3B), for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 (orange), or 900 (yellow) peak PAR µmol photons m^−2^s^−1^ (µE); and photoperiods of 8 (square), 12 (circle), 16 (triangle), or 24 (diamond) h. Solid blue line shows a fit of the pooled growth rates through peak PAR for each strain, with a three parameter model (Harrison and Platt, 1986). We also fit the same model separately for 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 together with 900 (orange) peak PAR µmol photons m^−2^s^−1^, only when they were each significantly different (ANOVA, *p* < 0.05) from the fit of pooled data. (**B**) Alpha parameters of the initial rise of growth rate (α) vs. cumulative diel Photosynthetically Usable Radiation (PUR), estimated from data pooled for each peak PAR (points (± SE) connected by dashed lines), and estimated for all data across all peak PAR, for each strain (solid blue horizontal line ± SE).", .sep = " ")
knitr::include_graphics(file.path(FiguresFP,"Fig_GrowthRate_Light_PUR.png"))
```

<br>

```{r GrowthRatePhotoperiodPAR, fig.cap = GrowthRatePhotoperiodPAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}
GrowthRatePhotoperiodPAR_cap<- glue("(**A**) Chlorophyll-specific exponential growth rates (d^−1^) vs. cumulative diel PAR (µmol photons m^−2^d^−1^). Growth rates (± SE falling within symbols) were estimated from logistic fits of chlorophyll proxy OD~680~ – OD~720~ (ΔOD) vs. elapsed time (Fig. 1, Fig. S3B), for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 (orange), or 900 (yellow) peak PAR µmol photons m^−2^s^−1^ (µE); and photoperiods of 8 (square), 12 (circle), 16 (triangle), or 24 (diamond) h. Solid blue line shows a fit of the pooled growth rates through photoperiod (h) for each strain, with a three parameter model [@harrisonPhotosynthesisirradianceRelationshipsPolar1986]. We also fit the same model separately for 8 (dotted line), 12 (long dash line), 16 (dashed line), or 24 (two dash line) h photoperiods, since for all strains they were each significantly different (ANOVA, *p* < 0.05) from the fit of pooled data. (**B**) Alpha parameters of the initial rise of growth rate (α) vs. cumulative diel PAR, estimated from data pooled for each photoperiod (points (± SE) connected by dashed lines), and estimated for all data across photoperiods (solid blue horizontal line ± SE), for each strain.", .sep = " ")
knitr::include_graphics(file.path(FiguresFP,"SFig_GrowthRate_Photoperiod_PAR.png"))
```

<br>

```{r GrowthRateLightPAR, fig.cap = GrowthRateLightPAR_cap, fig.height = 8, fig.width = 6, echo = FALSE}
GrowthRateLightPAR_cap<- glue("(**A**) Chlorophyll-specific exponential growth rates (d^−1^) vs. cumulative diel PAR (µmol photons m^−2^d^−1^). Growth rates (± SE falling within symbols) were estimated from logistic fits of chlorophyll proxy OD~680~ – OD~720~ (ΔOD) vs. elapsed time (Fig. 1, Fig. S3B), for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 (orange), or 900 (yellow) peak PAR µmol photons m^−2^s^−1^ (µE); and photoperiods of 8 (square), 12 (circle), 16 (triangle), or 24 (diamond) h. Solid blue line shows a fit of the pooled growth rates through peak PAR for each strain, with a three parameter model (Harrison and Platt, 1986). We also fit the same model separately for 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 together with 900 (orange) peak PAR µmol photons m^−2^s^−1^, only when they were each significantly different (ANOVA, *p* < 0.05) from the fit of pooled data. (**B**) Alpha parameters of the initial rise of growth rate (α) vs. cumulative diel PAR, estimated from data pooled for each peak PAR (points (± SE) connected by dashed lines), and estimated for all data across all peak PAR, for each strain (solid blue horizontal line ± SE).", .sep = " ")
knitr::include_graphics(file.path(FiguresFP,"SFig_GrowthRate_Light_PAR.png"))
```

<br>

```{r PhycoChlaRatio, fig.cap = PhycoChlaRatio_cap, fig.height = 6, fig.width = 8, echo = FALSE}
PhycoChlaRatio_cap<- glue("Changes of Phycobiliprotein to Chl *a* ratio (µg:µg) vs. cumulative diel PAR (µmol photons m^−2^d^−1^). Phycobiliprotein to Chl *a* ratio was estimated for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 (orange), or 900 (yellow) peak PAR µmol photons m^−2^s^−1^ (µE); and photoperiods of 8 (square), 12 (circle), 16 (triangle), or 24 (diamond) h. Figure presents data (smaller symbols) and means (bigger symbols) from exponential or pre-stationary phase of growth. Blue solid line shows single phase exponential decay fit for data from each strain and growth phase, fit parameters are presented. Different lowercase letters indicate statistically significant differences between the fit models for different strains within a given phase of growth. Different uppercase letters indicate statistically significant differences between the fit models for different phases of growth within a given strain (ANOVA; *p* < 0.05).", .sep = " ")
knitr::include_graphics(file.path(FiguresFP,"SFig_PhycoChlaRatio.png"))
```

<br>

```{r Sigma445, fig.cap = Sigma445_cap, fig.height = 8, fig.width = 6, echo = FALSE}
Sigma445_cap<- glue("Effective absorption cross section of PSII (\u03C3~PSII~′; nm^2^ quanta^−1^) measured under diel peak PAR growth light under blue (Ex~445nm~) excitation vs. cumulative diel PAR (µmol photons m^−2^d^−1^). \u03C3~PSII~′ was estimated for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 (orange), or 900 (yellow) peak PAR µmol photons m^−2^s^−1^ (µE); and photoperiods of 8 (square), 12 (circle), 16 (triangle), or 24 (diamond) h. Figure presents data (smaller symbols) and means (bigger symbols) from exponential or pre-stationary phase of growth. Blue solid line shows linear model fit for data from each strain and growth phase. Different lowercase letters indicate statistically significant differences between the fit models for different strains within a given phase of growth. Different uppercase letters indicate statistically significant differences between the fit models for different phases of growth within a given strain (*t*-test; *p* < 0.05).", .sep = " ")
knitr::include_graphics(file.path(FiguresFP,"SFig_Sigma445.png"))
```

<br>

```{r SigmaPig445, fig.cap = SigmaPig445_cap, fig.height = 8, fig.width = 6, echo = FALSE}
SigmaPig445_cap<- glue("(**A**) Changes of effective absorption cross section of PSII (\u03C3~PSII~; nm^2^ quanta^−1^) measured at the dark period under orange (Ex~590nm~) excitation vs. the ratio of sum of µg phycobilins (PE, PC, APC protein, Phycobiliprotein) to µg Chl *a*. (**B**) Changes of effective absorption cross section of PSII (\u03C3~PSII~′; nm^2^ quanta^−1^) measured under diel peak PAR growth light under blue (Ex~445nm~) excitation vs. the ratio of sum of µg phycobilins (PE, PC, APC protein, Phycobiliprotein) to µg Chl *a*. \u03C3~PSII~′ was estimated for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 (orange), or 900 (yellow) peak PAR µmol photons m^−2^s^−1^ (µE); and photoperiods of 8 (square), 12 (circle), 16 (triangle), or 24 (diamond) h. Figure presents data (smaller symbols) and means (bigger symbols) from exponential or pre-stationary phase of growth. Blue solid line shows linear model fit for data from each strain and growth phase. Different lowercase letters indicate statistically significant differences between the fit models for different strains within a given phase of growth. Different uppercase letters indicate statistically significant differences between the fit models for different phases of growth within a given strain (*t*-test; *p* < 0.05).", .sep = " ")
knitr::include_graphics(file.path(FiguresFP,"SFig_SigmaPigMerge.png"))
```

<br>

```{r GrowthRateJVPSIILight, fig.cap = GrowthRateJVPSIILight_cap, fig.height = 8, fig.width = 6, echo = FALSE}
GrowthRateJVPSIILight_cap<- glue("(**A**) Chlorophyll specific exponential growth rates (d^−1^) vs. cumulative diel PSII electron flux (*JV*~PSII~; µmol e^−^ µmol Chl *a*^−1^ d^−1^) measured under diel peak PAR growth light. Growth rates (± SE falling within symbols) were estimated from logistic fits of chlorophyll proxy OD~680~ – OD~720~ (ΔOD) vs. elapsed time (Fig. S3). PSII flux was estimated using FRRf induction curves with excitation of chlorophyll (Ex~445nm~, blue), for two PC-rich cultures (056; dark green, 077; light green) and two PE-rich cultures (048; light red, 127; dark red) of *Synechococcus* sp. grown at 30 (dark gray), 90 (light gray), 180 (purple), 300 (red), 600 (orange), or 900 (yellow) peak PAR µmol photons m^−2^s^−1^ (µE); and photoperiods of 8 (square), 12 (circle), 16 (triangle), or 24 (diamond) h. Solid blue line shows a fit of the pooled growth rates for each strain, with a three parameter model [@harrisonPhotosynthesisirradianceRelationshipsPolar1986]. We also fit the same model separately for 600 together with 900 (orange) peak PAR µmol photons m^−2^s^−1^, when they were significantly different (ANOVA, *p* < 0.05) from the fit of pooled data. (**B**) Alpha parameters of the initial rise of growth rate (α) vs. cumulative diel *JV*~PSII~, estimated from data pooled for each peak PAR (points (± SE) connected by dashed lines), and estimated for all data across all peak PAR, for each strain (solid blue horizontal line ± SE).", .sep = " ")
knitr::include_graphics(file.path(FiguresFP,"SFig_GrowthRate_JVPSII_Light_Chla.png"))
```

<br>

# References {.unnumbered}

